src/utils/SupabaseActions.ts

```diff
@@ -29,7 +29,7 @@ export class SupabaseActions extends SupabaseService {
       .from(supabaseBucket)
       .uploadToSignedUrl(signedUrl.path, signedUrl.token, file)
     if (error) {
-      throw new APIError(httpStatus.BAD_REQUEST, error.message)
+      throw new APIError(httpStatus.BAD_REQUEST, 'could not upload file to supabase', error.message)
```

Comment:
SupabaseActions is called from a client components so using APIError doesn't work here. Instead you should manually `console.error` the error without throwing an exception - since throwing an error will crash the app completely.